services/src/main/java/org/keycloak/services/managers/AuthenticationManager.java

```diff
@@ -909,7 +909,6 @@ public static AuthResult authenticateIdentityCookie(KeycloakSession session, Rea
         AuthResult authResult = verifyIdentityToken(session, realm, session.getContext().getUri(), session.getContext().getConnection(), checkActive, false, null, true, tokenString, session.getContext().getRequestHeaders(), VALIDATE_IDENTITY_COOKIE);
         if (authResult == null) {
             expireIdentityCookie(realm, session.getContext().getUri(), session);
-            expireOldIdentityCookie(realm, session.getContext().getUri(), session);
```

Comment:
I'm still seeing some set-cookie headers for the old paths without the '/' at the end.

Reply:
The "old" cookie is set within the `expireIdentityCookie` method. I'm only removing this unnecessary call.

Reply:
I'd vote for removing everything around the old path in this PR, as I think that's perfectly safe since we haven't set a cookie with that old path for years.

Reply:
I would love to remove it, but we should do it carefully as we might also review any test around it. I'm more in favor of a follow-up, specifically to remove this behavior.

Reply:
Ok, works for me